spec/classes/pulp_admin_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe 'pulp::admin' do
+  context 'on RedHat' do
+    context 'install class without parameters' do
+      it { should contain_class('pulp::admin::install') }
+      it { should contain_class('pulp::admin::config') }
+      it { should contain_class('pulp::admin::params') }
+
+      it { should contain_package('pulp-admin-client').with_ensure('installed') }
+
+      it 'should set admin.conf file' do
+        should contain_file('/etc/pulp/admin/admin.conf').
+        with_content(/^\[server\]$/).
```

Comment:
For readability I'd indent these with_ statements. For future reference you could also have a look at [verify_contents](https://github.com/puppetlabs/puppetlabs_spec_helper/blob/2c4550df345c40fb5b7f559b045bb03ee4e98611/lib/puppetlabs_spec_helper/module_spec_helper.rb#L9-L12):
```ruby
verify_contents(catalogue, '/etc/pulp/admin/admin.conf', ['line1', 'line2'])
```

Note the catalogue part is rspec-puppet 2.0, it was something else in rspec-puppet 1.x.